_posts/2024-02-20-NOAA-pkgdown.md

```diff
@@ -66,7 +66,19 @@ Want your pkgdown site to always be up to date with your code? Use the [`use_upd
 
 Optional steps to improve your pkgdown page
 
-### Adding a video tutorial to your landing page
+### Adding a hex logo to your pkgdown landing page
+
+To add a hex sticker to your pkgdown page:
+
+- Add a logo file to `man/figures/logo.png`. Make sure it's just named `logo.png`.
+- Then to the top of your README.md file where you normally have your package title, switch out the title line with this:
+```
+# yourpackagename <img src="man/figures/logo.png" align="right" width=250/>
```

Comment:
maybe edit to include alt text?

``` md
# yourpkgname <img src="man/figures/logo.png" alt="logo for the yourpkgname R package" align="right" width=250/>
```

I'm also questioning whether the `/` after width is intentional or not. Is it needed?

Reply:
I think that slash is necessary because it's an HTML chunk and the `/>` closes the chunk. I will add alt text; that is a good idea!

Reply:
I've added the alt text but I haven't actually revised a pull request before so I think this will work, but I'm not 100% sure... stand by.